dockerfiles/centos-7-pg10/Dockerfile

```diff
@@ -39,6 +39,10 @@ RUN ( [[ -z "epel-release centos-release-scl-rh" ]] || yum install -y epel-relea
         rpmlint \
         spectool \
         tar \
+        lz4 \
+        lz4-devel \
+        libzstd \
+        libzstd-devel \
```

Comment:
Why are we installing both x & x-devel packages ? Also, ol7 seems to be broken, can you check why ?

Reply:
Both files required. I have Niles checked.

Reply:
strictly speaking the [devel packages depend](https://packages.debian.org/buster/liblz4-dev) on the non devel packages, so they are not strictly required.

However I think it will be more readable between the container and the packages to include them both.